vmm/src/device_manager/mmio.rs

```diff
@@ -108,18 +112,14 @@ impl MMIODeviceManager {
             let io_addr = IoeventAddress::Mmio(
                 self.mmio_base + u64::from(devices::virtio::NOTIFY_REG_OFFSET),
             );
-            self.vm_requests.push(VmRequest::RegisterIoevent(
-                queue_evt.try_clone().map_err(Error::CloneIoEventFd)?,
```

Comment:
Looks like `CloneIoEventFd` and `CloneIrqFd` aren't used anywhere anymore, you can delete them.

Reply:
@aghecenco good catch, done.